easybuild/easyconfigs/m/mrsFAST/mrsFAST-Ultra-3.3.1-intel-compiler.patch

```diff
@@ -0,0 +1,52 @@
+diff -ru mrsfast-ultra-3.3.1.orig/Makefile mrsfast-ultra-3.3.1/Makefile
```

Comment:
@pescobar please add a line of comment explaining the patch, and add yourself as an author